apps/ledger-live-desktop/src/renderer/families/evm/StakeFlowModal/EthStakingModalBody.tsx

```diff
@@ -2,23 +2,30 @@ import { Flex, Text } from "@ledgerhq/react-ui";
 import { Account } from "@ledgerhq/types-live";
 import React, { useCallback, useState } from "react";
 import { useHistory } from "react-router-dom";
+import BigNumber from "bignumber.js";
+import { useTranslation } from "react-i18next";
 
 import { appendQueryParamsToDappURL } from "@ledgerhq/live-common/platform/utils/appendQueryParamsToDappURL";
-import { useTranslation } from "react-i18next";
+import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
 import { track } from "~/renderer/analytics/segment";
 import CheckBox from "~/renderer/components/CheckBox";
 import EthStakeIllustration from "~/renderer/icons/EthStakeIllustration";
-import { openURL } from "~/renderer/linking";
+
 import {
   CheckBoxContainer,
   LOCAL_STORAGE_KEY_PREFIX,
 } from "~/renderer/modals/Receive/steps/StepReceiveStakingFlow";
 import { ListProvider, ListProviders } from "./types";
 import { getTrackProperties } from "./utils/getTrackProperties";
 
-import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
 import ProviderItem from "./component/ProviderItem";
 
+const ETH_LIMIT = BigNumber(32).times(BigNumber(10).pow(18));
```

Comment:
Hi, shouldn't we use the `min` number coming from the feature flag here instead of the hardcoded `32` ?
Also, shouldn't we use the `magnitude` property from the ethereum cryptocurrency instead of the hardcoded `18` ?

Reply:
Yes I see where you're coming from. I've updated the eth magnitude to come from the currency info. However, I'm not sure if the firebase config is a particularly stable source of truth for the min. Ethereum validator stake, as it's just updated manually. The min value is really just boolean-like. It's either present (and therefore happens to be the constant, `32` ETH), or not present at all (hence the fallbacks to `0`). So if that minimum validator limit changes for the EVM chain as a whole, or if it varied per staking provider, then there would probably be several other changes to how we display e.g. the order of items in this component.